cleo5_extensions.sbf

```diff
@@ -1,6 +1,8 @@
+audio
 bitwise
 debug
 file
 ini
 memory
-text
\ No newline at end of file
+text
+math
```

Comment:
why new line here? Not alphabetical order?

Reply:
math is the new CLEO5 extension.
order does not matter